_posts/samus_returns/2024-04-22-export-romfs.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
anything special one has to do for the exheader later for MW?

Reply:
I'll worry about that later since it's not final nor relevant at the moment.